cpp/include/cudf/null_mask.hpp

```diff
@@ -112,9 +112,28 @@ void set_null_mask(bitmask_type* bitmask,
                    bool valid,
                    rmm::cuda_stream_view stream = cudf::get_default_stream());
 
+/**
+ * @brief Sets a vector of pre-allocated bitmask buffers to given states in the corresponding ranges
+ * in bulk
+ *
+ * Sets `[begin_bit, end_bit)` bits of each given bitmasks to corresponding valid state. This
```

Comment:
minor suggestion: This should say something about whether or not the `bitmasks` are allowed to alias. Consider the degenerate case where I pass in `32` bitmasks of a single bit each that all point into the same 32-bit word.

Reply:
Good suggestion. Updated the docstring